### PR TITLE
BREAKING: junit: Rename *SeedCorpus to *Inputs

### PR DESCRIPTION
**Pull request by fmeum**
The new name doesn't use fuzzing terminology and is more fitting now
that we also write crashing inputs into this directory.

### CODE REVIEW

**Comment by fmeum** [comment]
@bertschneider Stacked on #494 so that we get two changelog entries.

File: examples/junit/src/test/java/com/example/DirectoryInputsFuzzTest.java [hunk 1]
```diff
@@ -18,10 +18,10 @@
 import com.code_intelligence.jazzer.api.FuzzerSecurityIssueMedium;
 import com.code_intelligence.jazzer.junit.FuzzTest;
 
-public class DirectorySeedCorpusFuzzTest {
+public class DirectoryInputsFuzzTest {
   private static long runs = 0;
 
-  @FuzzTest(seedCorpus = "/com/example/DirectoryBasedSeedCorpus", maxDuration = "0s")
+  @FuzzTest(maxDuration = "0s")
   public void seedCorpusFuzz(FuzzedDataProvider data) {
```
Comment (bertschneider) [inline examples/junit/src/test/java/com/example/DirectoryInputsFuzzTest.java:25]: Rename test method as well?

File: driver/src/main/java/com/code_intelligence/jazzer/junit/JazzerFuzzTestExecutor.java [hunk 2]
```diff
@@ -78,47 +78,37 @@ public TestExecutionResult execute() throws IOException, URISyntaxException {
     // libFuzzer and also emit findings into it so that the regression test can be used to debug
     // them.
     FuzzTest fuzzTest = AnnotationSupport.findAnnotation(fuzzTestMethod, FuzzTest.class).get();
-    String seedCorpusResourcePath = fuzzTest.seedCorpus().isEmpty()
-        ? Utils.defaultSeedCorpusPath(fuzzTestClass)
-        : fuzzTest.seedCorpus();
+    String seedCorpusResourcePath = Utils.inputsDirectoryResourcePath(fuzzTestClass);
```
Comment (bertschneider) [inline driver/src/main/java/com/code_intelligence/jazzer/junit/JazzerFuzzTestExecutor.java:81]: Rename variable?

File: driver/src/main/java/com/code_intelligence/jazzer/junit/FuzzTest.java [hunk 3]
```diff
@@ -50,17 +50,6 @@
 // from parallelization.
 @ResourceLock(value = "jazzer", mode = ResourceAccessMode.READ_WRITE)
 public @interface FuzzTest {
```
Comment (bertschneider) [inline driver/src/main/java/com/code_intelligence/jazzer/junit/FuzzTest.java:52]: JavaDoc still mentions `seedCorpus`.

**Review by bertschneider — APPROVED** [review]
I like the new name :+1:

File: driver/src/main/java/com/code_intelligence/jazzer/junit/FuzzTest.java (hunk 3 above)
Comment (fmeum) [reply driver/src/main/java/com/code_intelligence/jazzer/junit/FuzzTest.java:49]: Removed that part, the docs will have to be rewritten anyway.

File: driver/src/main/java/com/code_intelligence/jazzer/junit/JazzerFuzzTestExecutor.java (hunk 2 above)
Comment (fmeum) [reply driver/src/main/java/com/code_intelligence/jazzer/junit/JazzerFuzzTestExecutor.java]: Done, also updated the comments and other variables.

File: examples/junit/src/test/java/com/example/DirectoryInputsFuzzTest.java (hunk 1 above)
Comment (fmeum) [reply examples/junit/src/test/java/com/example/DirectoryInputsFuzzTest.java]: Done.

**Pull request merged**